### ISSUE TITLE
Configure overcommit_ratio on PostgreSQL servers

### ISSUE THREAD

**Issue by RobHooper** [open-contracting/deploy#524]
unattended-upgrades on ocp22 has been failing reporting it is unable to allocate enough memory.
We have temporarily set `overcommit_memory` = `0` (the default value) to mitigate this.

PostgreSQL recommends `overcommit_memory` = `2` to help protect the service from accidental termination when memory resources are exhausted, however, this setting limits memory for all process causing issues particularly on smaller servers.

To resolve this we should increase `overcommit_ratio` from 50% to 90% / 100% allowing user space to access all available memory.

Further reading:
* https://www.postgresql.org/docs/current/kernel-resources.html#LINUX-MEMORY-OVERCOMMIT
* https://www.kernel.org/doc/Documentation/vm/overcommit-accounting

**Comment by RobHooper** [comment]
PostgreSQL on Ubuntu automatically adjusts `oom_score_adj` as mentioned in the PostgreSQL docs as an alternative approach to this problem. Reducing `oom_score_adj` prevents Out of Memory killer from targeting the service.
Therefore we can prioritise setting `overcommit_memory=1` following the Redis guidelines.

https://serverfault.com/a/757666

**Comment by jpmckinney** [comment]
Just checking - the issue description was about setting `overcommit_ratio`, but #545 only change `overcommit_memory` and `swappiness`. Is that sufficient, or do we still want to set `overcommit_ratio`?

**Comment by RobHooper** [comment]
@jpmckinney , You are quite right, `overcommit_ratio` was missed. I have created a PR correcting this (#546).
